### ISSUE TITLE
Bug with visualization

### ISSUE THREAD

**Issue by mpelchat04** [NRCan/geo-deep-learning#115]
If `vis_batch_range` parameter in config file is left empty, the process will crash. 
Here's the error:
```
Epoch 0/49
--------------------
Traceback (most recent call last):
  File "/gpfs/fs2/nrcan/geobase/transfer/work/deep_learning/lidar/test_RGBN_temp/geo-deep-learning/train_segmentation.py", line 646, in <module>
    main(params, config_path)
  File "/gpfs/fs2/nrcan/geobase/transfer/work/deep_learning/lidar/test_RGBN_temp/geo-deep-learning/train_segmentation.py", line 354, in main
    debug=debug)
  File "/gpfs/fs2/nrcan/geobase/transfer/work/deep_learning/lidar/test_RGBN_temp/geo-deep-learning/train_segmentation.py", line 466, in train
    min_vis_batch, max_vis_batch, increment = vis_batch_range
TypeError: 'NoneType' object is not iterable
```

**Comment by mpelchat04** [comment]
Same goes at validation time: 
```
  File "/gpfs/fs2/nrcan/geobase/transfer/work/deep_learning/lidar/test_RGBN_temp/geo-deep-learning/train_segmentation.py", line 648, in <module>
    main(params, config_path)
  File "/gpfs/fs2/nrcan/geobase/transfer/work/deep_learning/lidar/test_RGBN_temp/geo-deep-learning/train_segmentation.py", line 371, in main
    debug=debug)
  File "/gpfs/fs2/nrcan/geobase/transfer/work/deep_learning/lidar/test_RGBN_temp/geo-deep-learning/train_segmentation.py", line 539, in evaluation
    min_vis_batch, max_vis_batch, increment = vis_batch_range
TypeError: 'NoneType' object is not iterable
```

**Comment by mpelchat04** [comment]
Closed by #133
